toolchain/check/merge.h

```diff
@@ -7,22 +7,25 @@
 
 #include "toolchain/check/context.h"
 #include "toolchain/sem_ir/ids.h"
+#include "toolchain/sem_ir/import_ir.h"
 
 namespace Carbon::Check {
 
-struct PrevInstForMerge {
+struct InstForMerge {
   // The resolved instruction.
   SemIR::Inst inst;
-  // Whether an import was encountered during resolution.
-  bool is_import;
+  // The imported instruction, or invalid if not an import. This should
+  // typically only be used for the ImportIRId, but we only resolve that if
+  // needed.
```

Comment:
I don't understand what the part of this second sentence after the comma is telling me. What do we only resolve if needed -- the ImportIRId? If so, what does it mean to resolve an ImportIRId?

Reply:
rephrasing: resolve that -> load it